documentation/get-started/concepts.en.md

```diff
@@ -1,46 +1,37 @@
 ---
-title: Concepts
+title: Principles
 tx_slug: documentation_get-started_concepts
 ---
 
-# Concepts
+# Principles
 
-QField was designed with a few key concepts in mind.
+QField is designed with a few key principles in mind.
 
-## Keep it simple
 
-The requirements on the field are not the same as on a desktop. The
-screen is smaller, the input devices are different and the tasks are
-different.
+## Keeping it simple
 
-QField aims to help users to perform the tasks they need to do without
-cluttering the user interface. This means, that only tasks which need to
-be done on the field are availble from the interface. Everything else is
-not.
+The requirements in the field are not the same as on a desktop.
+Firstly, when using a phone or a tablet, the canvas to work on is much more limited than a computer monitor.
+Secondly, devices for data collection as well as the individual tasks that need to be performed are vastly different.
 
-This means that everything like layer styling, form definitions and
-other project setup steps should be done on a computer with QGIS
-installed first.
+QField aims to help users do fieldwork without the nuisance of a cluttered user interface.
+In other words, only the relevant parts of a task are accessible on the interface, while less important things remain out of sight.
 
-## Be compatible with QGIS
+Before heading out to the field though, preliminary steps such as layer styling, tailored forms, or any other project configurations should first be done on the QGIS desktop version.
 
-QField is based on QGIS. It is not a rebuild of QGIS it really *does*
-use QGIS libraries. The rendering engine is exactly the same as in QGIS
-for desktop and your project will therefore look exactly the same on
-your mobile device as it does on your computer.
+## Compatibility with QGIS
 
-If something is already available as a configuration option in a QGIS
-project, it should not be re-invented. QField therefore uses the same
-edit widgets as QGIS desktop does. If a project is already configured
-for the desktop, it should just run on mobile as well.
+Since QField relies on existing QGIS Libraries, it is not an extensive rebuild of QGIS. 
+The rendering engine in QField is the same as the one used in desktop QGIS, ensuring that projects will appear virtually identical in both environments.
```

Comment:
I believe this can be rephrased in another way, especially the first sentence is super technical.

What if we say for a first sentence:
> Since QField is a mobile optimized version of the QGIS desktop app, your project will appear and feel identical both in QGIS and QField.

Reply:
Yes, I agree on that one too. Seems more user-friendly.